components/network/ssh-askpass/network-ssh-askpass.p5m

```diff
@@ -0,0 +1,35 @@
+#
```

Comment:
Hi, I'm back from vacation. Nice of you to roam through so many PRs while I was away ;p

As for several packages - it is an option which is possible, and may be attractive to some people. For example, if x11-ssh-askpass worked here (I didn't bother too much trying to make it build), it could be set up to securely block the X11 session until you enter the passphrase. I guess some people like that; for me the simple zenity requestor suffices. And extending the way to set up probable multiple packages later on, instead of preparing now, seems like bad foresight.